src/components/Menu.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+// Navbar component
+const Navbar = () => {
+  const menuItems = {
+    home: "Home",
+    calculator: "Calculator",
+    quote: "Quote",
+  };
+  return (
+    <div className="d-inline-flex px-3 w-100 bg-light">
+      <h5>Math Magicians</h5>
+      <ul className="d-flex justify-content-end gap-5 w-100 mt-2">
+        {Object.keys(menuItems).map((item) => (
+          <li key={item} className="list-group-item">
+            <a href={`${item}`} className="text-decoration-none text-dark">
+              {menuItems[item]}
+            </a>
+          </li>
```

Comment:
- [ ] Your app refreshes when I navigate across the pages. That should not be as this is a single-page app. Meaning we should be able to navigate the app without reloading it. You should consider using [NavLink from the `react-router-dom`](https://reactrouter.com/en/main/components/nav-link) package.

Reply:
@DammyShittu Thank you for your guidance and kind review